packages/@lwc/synthetic-shadow/src/faux-shadow/element.ts

```diff
@@ -379,18 +347,12 @@ defineProperties(Element.prototype, {
                 elementQuerySelectorAll.apply(this, ArraySlice.call(arguments) as [string])
             );
 
-            if (!featureFlags.ENABLE_NODE_LIST_PATCH) {
-                const filteredResults = getFilteredArrayOfNodes(
-                    this,
-                    nodeList,
-                    ShadowDomSemantic.Disabled
-                );
-                return createStaticNodeList(filteredResults);
-            }
-
-            return createStaticNodeList(
-                getFilteredArrayOfNodes(this, nodeList, ShadowDomSemantic.Enabled)
+            const filteredResults = getFilteredArrayOfNodes(
```

Comment:
now that this function is never called with `ShadowDomSemantic.Enabled` as 3rd argument, should we remove that argument and the code that now is unreachable?

Reply:
Good catch! Removed the enum entirely.